boards/raspberrypi/rpi_pico2/rpi_pico2.yaml

```diff
@@ -1,5 +1,5 @@
-identifier: rpi_pico2
-name: Raspberry Pi Pico 2
+identifier: rpi_pico2/rp2350a/m33
```

Comment:
these changes need to be part of the previous commits, you don't add a commit which is wrong then add fixed to it in another commit

Reply:
I completely agree with @nordicjm on this.
Reviewing a PR, commit by commit, making pending comments, just to discover that n-commits later what I spent time on reviewing and writing a comment about is now changed is very annoying.